docs/user_guide.rst

```diff
@@ -45,11 +45,32 @@ After installation Qiskit on IQM can be imported in your Python code as follows:
    from iqm import qiskit_iqm
 
 
-Running a quantum circuit on an IQM quantum computer
-----------------------------------------------------
+Authentication
+--------------
 
-In this section we demonstrate the practicalities of using Qiskit on IQM on an example of constructing and executing
-a simple quantum circuit on an IQM quantum computer.
+If the IQM server you are connecting to requires authentication, you may use
+`Cortex CLI <https://github.com/iqm-finland/cortex-cli>`_ to retrieve and automatically refresh access tokens,
+then set the :envvar:`IQM_TOKENS_FILE` environment variable, as instructed, to point to the tokens file.
+See Cortex CLI's `documentation <https://iqm-finland.github.io/cortex-cli/readme.html>`__ for details.
+
+If you are using ``IQM Resonance``, authentication is handled differently.
+Use the :envvar:`IQM_TOKEN` environment variable to provide the API Token obtained
+from the server dashboard.
+
+Finally, you may authenticate yourself using the :envvar:`IQM_AUTH_SERVER`,
+:envvar:`IQM_AUTH_USERNAME` and :envvar:`IQM_AUTH_PASSWORD` environment variables, or pass them as
+arguments to :class:`.IQMProvider`, however this approach is less secure and considered deprecated.
```

Comment:
Perhaps it makes sense to swap these two paragraphs. The last paragraph is an alternative to non-Resonance way described earlier. The paragraph about `IQM_TOKEN` describes the way which differs from both non-Resonance ways.

Reply:
With a direct Cocos access you can use any means of authentication, right?
It's just Resonance that only supports IQM_TOKEN.

Reply:
I reworded it.